heroes.js

```diff
@@ -20,25 +19,25 @@ function createHeroBlock(hero) {
 
 function setupHeroReveal() {
   for (var i = 0; i < heroes.length; i++) {
-    var hero = heroes[i]
+    const hero = heroes[i]
```

Comment:
Using const here means that hero is scoped to the for loop, not the function. That means each iteration of the loop gets its own version of hero instead of each iteration overwriting the same variable.

The other option would be to use a forEach since the inner function will also have its own scope.